proxy/logstats.cc

```diff
@@ -594,15 +594,16 @@ struct CommandLineArgs {
   int64_t min_hits;
   int max_age;
   int line_len;
-  int incremental; // Do an incremental run
-  int tail;        // Tail the log file
-  int summary;     // Summary only
-  int json;        // JSON output
-  int cgi;         // CGI output (typically with json)
-  int urls;        // Produce JSON output of URL stats, arg is LRU size
-  int show_urls;   // Max URLs to show
-  int as_object;   // Show the URL stats as a single JSON object (not array)
-  int concise;     // Eliminate metrics that can be inferred by other values
+  int incremental;      // Do an incremental run
+  int tail;             // Tail the log file
+  int summary;          // Summary only
+  int json;             // JSON output
+  int cgi;              // CGI output (typically with json)
+  int urls;             // Produce JSON output of URL stats, arg is LRU size
+  int show_urls;        // Max URLs to show
+  int as_object;        // Show the URL stats as a single JSON object (not array)
+  int concise;          // Eliminate metrics that can be inferred by other values
+  bool report_per_user; // A flag to aggregate and report stats per user instead of per host if 'true' (default 'false')
```

Comment:
I'm ok with this, but if we use bool here, we should use bool for the others too (that are bool). The reason I initially had it as "int" (i think) was because I assumed the parser needed the pointer to an int and not a bool. If it's safe and ok to use bool there, we should switch consistently. So, these

```
  -A, --as_object         tog   false     Produce URL stats as a JSON object instead of array
  -C, --concise           tog   false     Eliminate metrics that can be inferred from other values
  -i, --incremental       tog   false     Incremental log parsing
  -s, --summary           tog   false     Only produce the summary
  -j, --json              tog   false     Produce JSON formatted output
  -c, --cgi               tog   false     Produce HTTP headers suitable as a CGI
  -r, --report_per_user   tog   false     Report stats per user instead of host
```